scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -713,11 +888,12 @@
                (-<>> (concat running-task-ents pending-task-ents)
                      (group-by util/task-ent->user)
                      (map (fn [[user task-ents]]
-                            [user (into (sorted-set-by util/same-user-task-comparator) task-ents)]))
+                            [user (into (sorted-set-by (util/same-user-task-comparator)) task-ents)]))
                      (into {})
                      (dru/sorted-task-scored-task-pairs <> user->dru-divisors)
                      (filter (fn [[task scored-task]]
-                               (contains? pending-task-ents task)))
+                               (or (contains? pending-task-ents task)
+                                   (:instance/backfilled? task))))
```

Comment:
Can you put a comment why we include backfilled here? I want to make sure no one removed this by accident.